packages/cli/test/scripts/session.test.js

```diff
@@ -11,22 +11,22 @@ describe('session script', function () {
   const opts = { network: 'foo', from: '0x1', timeout: 10 }
 
   describe('opening a new session', function () {
-    describe('when there was no session opened before', function () {
-      describe('when the time out does not expire', function () {
+    context('when there was no session opened before', function () {
```

Comment:
Just curious: why this change?

Reply:
I know that `context` is a synonym for `describe`, but when talking about contexts (*when X is happening*) I prefer using context rather than `describe`. FYI, I started drafting an issue (pending to publish) with a list of conventions we should follow.